serving/docker/tensorrt-llm.Dockerfile

```diff
@@ -14,11 +14,11 @@ FROM nvidia/cuda:$version
 ARG cuda_version=cu122
 ARG python_version=3.10
 ARG TORCH_VERSION=2.1.2
-ARG djl_version=0.27.0~SNAPSHOT
+ARG djl_version=0.27.0
 ARG transformers_version=4.38.1
 ARG accelerate_version=0.27.0
 ARG tensorrtlibs_version=9.2.0.post12.dev5
-ARG trtllm_toolkit_version=nightly
+ARG trtllm_toolkit_version=0.8.0
```

Comment:
@xyang16 we can keep this as nightly. We're currently not pushing a versioned toolkit. This is a TO-DO (subject to some proper branch creation and tagging in the repo)

Reply:
Done